docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -281,6 +281,24 @@ We need to obtain and pass in a valid JWT to access that endpoint. There are two
 
 === Configuring the {extension-name} Extension Security Information
 
+[NOTE]
```

Comment:
@mcruzdev Usually, this kind of info would be added in the Testing section, the Configuring section is more about configuring properties in prod... But in any case, let's work on the docs once we finalize the solution